app/integration_test/app_test.dart

```diff
@@ -87,12 +87,12 @@ void main() {
 
       // We assume that the user is in at least 5 groups with the following
       // group names.
-      expect(find.text('10A'), findsOneWidget);
-      expect(find.text('Deutsch LK'), findsOneWidget);
-      expect(find.text('Englisch LK'), findsOneWidget);
-      expect(find.text('Französisch LK'), findsOneWidget);
-      expect(find.text('Latein LK'), findsOneWidget);
-      expect(find.text('Spanisch LK'), findsOneWidget);
+      await waitFor(tester, find.text('10A'));
```

Comment:
So one can't specify sth like `findsOneWidget` with the new method?

Reply:
Also couldn't we use the `waitFor` for only the first group name since the others are loaded by then? Otherwise it might take until we hit the timeout maybe several times if there are a few groups where the account is missing.


In this way there would also be clear separation that we `waitFor` the groups being loaded and then use `findsOneWidget` so that it is clear that we expect one group with the name for all the groups.

Reply:
We can first check if can find `Meine Klasse:`:

```dart
// Ensure that the group list is loaded. When the school class is loaded,
// we assume that the courses list is loaded as well.
await waitFor(tester, find.text('Meine Klasse:'));
```

and then test with `expect` again as we did before.